lib/sles4sap.pm

```diff
@@ -245,6 +245,7 @@ sub copy_media {
     # Then verify everything was copied correctly
     # NOTE: checksum is generated with this command: "find . -type f -exec md5sum {} \; > checksums.md5sum"
     my $chksum_file = 'checksum.md5sum';
+    return 1 if (!-f $chksum_file);
```

Comment:
This means that we return a OK if there is no checksum file? I disagree in that case.
If this is needed for test purpose it could be better to use a variable like `DISABLE_SAP_CHECK` of something like this.